tpl_lessallrounder/index.php

```diff
@@ -153,6 +153,11 @@
 				<span class="shadow-right">&nbsp;</span>
 				<?php require $modules . 'aftercontent.php'; ?>
 			</div>
+			<?php if (($this->countModules('position-8')) and ($params->get('rightSBtoBottom')) ) : ?>
```

Comment:
Same as above, use:
`if ($this->countModules('position-8') and $params->get('rightSBtoBottom', 1)) :`